### DISCUSSION TITLE
Beginner question about Post content and hooks

### DISCUSSION THREAD

**Discussion by eanyanwu** [wp-graphql/wp-graphql#2608] [Q&A] [answered]
Hello 👋🏽 

Does the following line:
https://github.com/wp-graphql/wp-graphql/blob/b89a31fd35e619b65b5dd85acaac182ed6fd61c5/src/Model/Post.php#L480

mean that all the installed plugins would get their `the_content` callbacks called before the post's content is returned?

Asked another way, should I expect the `data.post.content` in the response to have the post content with the modifications/transformations of all wordpress plugins that hook into `the_content`?

**Comment by justlevine** [comment, marked as answer]
hey @eanyanwu Yes, thats exactly what that line of code does :-)
